ArchiSteamFarm/Steam/Interaction/Commands.cs

```diff
@@ -81,11 +82,23 @@ public sealed class Commands {
 	}
 
 	[PublicAPI]
+	[Obsolete($"Use overload which accepts {nameof(EAccess)} instead, this one will be removed soon.", true)]
 	public async Task<string?> Response(ulong steamID, string message) {
 		if ((steamID == 0) || !new SteamID(steamID).IsIndividualAccount) {
 			throw new ArgumentOutOfRangeException(nameof(steamID));
 		}
 
+		EAccess access = Bot.GetAccess(steamID);
+
+		return await Response(access, message).ConfigureAwait(false);
+	}
+
+	[PublicAPI]
+	public async Task<string?> Response(EAccess access, string message, ulong steamID = 0) {
+		if (!Enum.IsDefined(typeof(EAccess), access)) {
```

Comment:
Why use `Enum.IsDefined(Type, object)` instead of `Enum.IsDefined<TEnum>(TEnum) where TEnum : struct, Enum`? It's much shorter (and simpler) to use it:
```diff
- if(!Enum.IsDefined(typeof(EAccess), access)) {
+ if(!Enum.IsDefined(access)) {
    ...
}
```

Reply:
No good reason, rather than convention with the rest of ASF. Not going to change it in this PR, but it's a very good idea for follow-up commit to change ASF convention into newer syntax - I'm 100% sure it wasn't always when I put the first enum checks.

Reply:
Yes. The generic overload was added with `net5.0` IIRC.